docs/book/pages/app_customization/bases_and_variants.md

```diff
@@ -30,6 +30,26 @@ Bases can be:
 - Relative paths from the `kustomization.yaml` - e.g. `../base`
 - Urls - e.g. `github.com/kubernetes-sigs/kustomize/examples/multibases?ref=v1.0.6`
 
+### Diagrams
+
+Single Base inheritted by single Application
+
+```mermaid
+graph TD;
+  B[Base Config]-->A[App];
+```
+
+Shared Bases inheritted by different Applications
+
+```mermaid
+graph TD;
+  B1[Base1 Config]-->A1[App 1];
+  B2[Base2 Config]-->A1[App 1];
+  B1[Base2 Config]-->A2[App 2];
+  B1[Base3 Config]-->A2[App 2];
+```
```

Comment:
These boxes are not large enough to show all the characters. See the diagrams

https://deploy-preview-618--kubectl-docs.netlify.com/pages/app_customization/bases_and_variants.html